python/lsst/ts/MTAlignment/component.py

```diff
@@ -149,15 +150,15 @@ def __init__(self):
                 if randrange(5) == 4:
                     cam_aligned = True
                     print("cam aligned")
-            asyncio.sleep(0.5)
+            await asyncio.sleep(0.5)
```

Comment:
why sleep for 0.5s here?

Reply:
removed this part of the code entirely since it doesn't really belong in the component/model